src/metrocs/redistricting/StarGeneratorTest.java

```diff
@@ -0,0 +1,32 @@
+package metrocs.redistricting;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.AnyOf.anyOf;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.ArrayList;
+/**
+ * Tests for Voter objects.
+ *
+ * @author Jada, David, Colin, John, Daniel, Ali
+ * @version
+ */
+public class StarGeneratorTest {
+    @Test
+    public void generatePatternTest() {
+        ArrayList<ArrayList<Location>> firstLocation = new ArrayList<ArrayList<Location>>();
+        ArrayList<ArrayList<Location>> secondLocation = new ArrayList<ArrayList<Location>>();
```

Comment:
Confusing variable names. The use of "...Location" suggests that the variable is holding a reference to a Location object, but that isn't the case.

What is the intent of the initializations? The values assigned don't appear to be used anywhere.